cylc/flow/config.py

```diff
@@ -155,7 +198,21 @@ def interpolate_template(tmpl, params_dict):
         raise ParamExpandError('bad template syntax')
 
 
-# TODO: separate config for run and non-run purposes?
+def dequote(s):
+    """Strip quotes off a string.
+
+    Examples:
+        >>> dequote('"foo"')
+        'foo'
+        >>> dequote("'foo'")
+        'foo'
+        >>> dequote('foo')
+        'foo'
+
+    """
+    if (s[0] == s[-1]) and s.startswith(("'", '"')):
+        return s[1:-1]
+    return s
```

Comment:
What about `dequote('')`? At the moment it raises an `IndexError`

Reply:
Added a return/test.